notify/opsgenie/opsgenie.go

```diff
@@ -120,7 +120,14 @@ func (n *Notifier) createRequest(ctx context.Context, as ...*types.Alert) (*http
 
 	tmpl := notify.TmplText(n.tmpl, data, &err)
 
-	details := make(map[string]string, len(n.conf.Details))
+	details := make(map[string]string)
+
```

Comment:
to be very strict this new line could be omitted. It's best to only put new line to separate logically different sections, whereas the map is related to below if. Not a blocker (: